### PR TITLE
Add cc option

### PR DESCRIPTION
**Pull request by davydovanton**
/cc @hanami/core-team @hanami/contributors

### CODE REVIEW

File: lib/hanami/mailer/dsl.rb [hunk 1]
```diff
@@ -139,6 +140,95 @@ def from(value = nil)
         end
       end
 
+      # Sets the cc (carbon copy) for mail messages
+      #
+      # It accepts a hardcoded value as a string or array of strings.
+      # For dynamic values, you can specify a symbol that represents an instance
+      # method.
+      #
+      # This value is not required.
```
Comment (joneslee85) [inline lib/hanami/mailer/dsl.rb]: What do you think about rephrase it to `This value is optional`?

File: lib/hanami/mailer/dsl.rb [hunk 2]
```diff
@@ -139,6 +140,95 @@ def from(value = nil)
...
+      #
+      # When a value is given, specify the cc of the email Otherwise,
```
Comment (joneslee85) [inline lib/hanami/mailer/dsl.rb]: seems to miss a dot after `email` and before `Otherwise`

**Comment by joneslee85** [comment]
@davydovanton thanks much 👍

**Comment by davydovanton** [comment]
@joneslee85 thanks for merge, I'll fix documentation n the next PR.

**Comment by jodosha** [comment]
@davydovanton Thank you! 👍  How about `bcc` too?

**Comment by davydovanton** [comment]
yep, I want to realise this today :)
